packages/babel-generator/src/generators/expressions.js

```diff
@@ -125,6 +125,10 @@ export function OptionalMemberExpression(node: Object) {
   }
 }
 
+export function Partial() {
```

Comment:
This name doesn't match with the name defined in the parser. Also, could you add a generator test?

Reply:
Sure, I'll add tests for the generator.
Do you mean in: `packages/babel-parser/src/types.js`
I see that I made a mistake there. I'll fix it 👍

Reply:
Additionally, I suppose, I can move this function to `src/generators/types.js`? Because we are not considering `Partial` (or if we call it `ArgumentPlaceholder`) to be an expression.

Reply:
Yeah good idea (I still prefer ArgumentPlaceholder)